guix-systole/packages/teem.scm

```diff
@@ -0,0 +1,38 @@
+(define-module (guix-systole packages teem)
+  #:use-module (guix-systole packages vtk)
+  #:use-module (guix build-system cmake)
+  #:use-module (guix download)
+  #:use-module (guix gexp)
+  #:use-module ((guix licenses)
+                #:prefix license:)
+  #:use-module (guix packages)
+  #:use-module ((guix-systole licenses)
+                #:prefix license:))
+
+(define-public teem-slicer
+  (package
+    (name "teem-slicer")
+    (version "1.12.0")
+    (source
+     (origin
+       (method url-fetch)
+       (uri
+        "https://github.com/Slicer/teem/archive/e4746083c0e1dc0c137124c41eca5d23adf73bfa.tar.gz")
+       (sha256
+        (base32 "0y8wwzkflj6v5nx0v8cgzryqlxii0px3mcgb3bff1nhyr5zf9yp1"))
+       (patches (list (local-file
```

Comment:
This could be fine like this. However there is some infrastructure in `packages.scm` to search patches. Whatever we do we should be consistent. Either we use this simple approach and remove the related code in `packages.scm` or we adopt that functionality here too.